wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/MessageSendFailureView.swift

```diff
@@ -108,22 +103,18 @@ final class MessageSendFailureView: UIView {
             NSLayoutConstraint.activate([
                 stackView.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 10.0),
                 stackView.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -10.0),
-                stackView.topAnchor.constraint(equalTo: topAnchor),
-                stackView.bottomAnchor.constraint(equalTo: bottomAnchor, constant: -8.0),
-
-                retryButton.topAnchor.constraint(equalTo: buttonContainer.topAnchor),
-                retryButton.bottomAnchor.constraint(equalTo: buttonContainer.bottomAnchor),
-                retryButton.trailingAnchor.constraint(equalTo: buttonContainer.trailingAnchor),
-                retryButton.leadingAnchor.constraint(greaterThanOrEqualTo: buttonContainer.leadingAnchor)
+                stackView.topAnchor.constraint(equalTo: topAnchor)
             ])
+            bottomConstraint = stackView.bottomAnchor.constraint(equalTo: bottomAnchor, constant: -8.0)
         } else {
             NSLayoutConstraint.activate([
                 stackView.leadingAnchor.constraint(equalTo: leadingAnchor),
                 stackView.trailingAnchor.constraint(equalTo: trailingAnchor),
-                stackView.topAnchor.constraint(equalTo: topAnchor),
-                stackView.bottomAnchor.constraint(equalTo: bottomAnchor)
+                stackView.topAnchor.constraint(equalTo: topAnchor)
             ])
+            bottomConstraint = stackView.bottomAnchor.constraint(equalTo: bottomAnchor)
         }
+        bottomConstraint?.isActive = true
```

Comment:
suggestion: Sync it already with the isHidden property 

```suggestion
        bottomConstraint?.isActive = !isHidden
```

Reply:
When i directly sync it with isHidden property, it is not updating correctly. Hence i added it inside override function